### DISCUSSION TITLE
Pimcore X Upgrade - Issue with hwi_oauth

### DISCUSSION THREAD

**Discussion by ShaileshT-dot** [pimcore/pimcore#9480] [Q&A / Support] [answered]
Hi Team,
We are upgrading our pimcore from 6.9 to pimcore X. We have followed the instructions as per [https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/Updating_Pimcore/V6_to_V10.html]. 
We are stuck at following error and are not able to proceed. Any suggestion would be of great help.

Script Pimcore\Composer::clearCache handling the pimcore-scripts event terminated with an exception

  [RuntimeException]
  An error occurred when executing the "/usr/bin/php8.0 --php-ini=/etc/php/8.0/cli/php.ini bin/console --ansi cache:clear --no-
  warmup" command:

  Exit code: 255

  Symfony\Component\Config\Definition\Exception\InvalidConfigurationException {#7603
    -path: "hwi_oauth"
    -containsHints: false
    #message: "The child config "firewall_names" under "hwi_oauth" must be configured."
    #code: 0
    #file: "./vendor/symfony/symfony/src/Symfony/Component/Config/Definition/ArrayNode.php"
    #line: 222

Notes:
In config/cmf.yaml, we have the following value
hwi_oauth:
    firewall_names: [demo_frontend]

In config/packages/security.yaml, we have values configured for "demo_frontend". we have configured it as per the [https://github.com/pimcore/demo/blob/10.x/config/packages/security.yaml](url)

**Comment by fashxp** [comment]
is your config/cmf.yml loaded properly?

**Reply by ShaileshT-dot** [reply]
I believe it is indeed loading well but I have not tested it. As part of Pimcore X migration, the file is now named config/cmf.yaml (not cmf.yml).
can you please suggest a way to check if it loads well?

**Comment by ShaileshT-dot** [comment, marked as answer]
The issue is resolved. The issue was that addtional folders and files within them were required, under config/packages folder,  which was missed. By referring to https://github.com/pimcore/demo/blob/10.x/config, we added the required fodlers/files and it  helped resolve.
Thanks for the support.
